core/apns/src/apns_connection.erl

```diff
@@ -119,8 +119,8 @@ open_out(Connection) ->
     ssl_opts(Connection),
     Connection#apns_connection.timeout
   ) of
-    {ok, OutSocket} -> {ok, OutSocket};
-    {error, Reason} -> {error, Reason}
+    {ok, _OutSocket} = OK -> OK;
+    {error, _Reason} = Er -> Er
```

Comment:
This was done by hand, in order to have the `->` aligned.  Tidier chooses variable names based on the tag of the structure (`Ok` and `Error` in this case), possibly with an appended number (e.g. `Error2`) if the variable is already used in the clause.